packages/cli/src/modules/data-table/data-store-rows.repository.ts

```diff
@@ -71,58 +76,55 @@ function getConditionAndParams(
 	};
 
 	if (operators[filter.condition]) {
-		return [
-			`${column} ${operators[filter.condition]} :${paramName}`,
-			{ [paramName]: filter.value },
-		];
+		return [`${column} ${operators[filter.condition]} :${paramName}`, { [paramName]: value }];
 	}
 
 	switch (filter.condition) {
 		// case-sensitive
 		case 'like':
 			if (['sqlite', 'sqlite-pooled'].includes(dbType)) {
-				const globValue = toSqliteGlobFromPercent(filter.value as string);
+				const globValue = toSqliteGlobFromPercent(value as string);
```

Comment:
(future refactoring / tech debt:) With better typing we wouldn't need these casts, as we already validate that hte conditions and data types match.